cmake/projects/EnTT/hunter.cmake

```diff
@@ -0,0 +1,29 @@
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+# Load used modules
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+include(hunter_cmake_args)
+
+hunter_add_version(
+    PACKAGE_NAME
+    EnTT
+    VERSION
+    3.0.0-WIP
```

Comment:
Please make it match git tag: `v3.x.y-WIP`

Reply:
Done, pushing shortly